daemon/src/main/scala/org/apache/gearpump/cluster/main/AppSubmitter.scala

```diff
@@ -38,7 +38,7 @@ object AppSubmitter extends App with ArgumentsParser {
     "namePrefix" -> CLIOption[String]("<application name prefix>", required = false, defaultValue = Some("")),
     "jar" -> CLIOption("<application>.jar", required = true))
 
-  def start : Unit = {
+  def main(akkaConf: Config, args: Array[String]): Unit = {
```

Comment:
any other way to do this ? Currently, users are blind about whether they have successfully submit their applications.